src/ec-evaluator/interpreter.ts

```diff
@@ -499,5 +521,13 @@ const cmdEvaluators: { [type: string]: CmdEvaluator } = {
     stash: Stash
   ) {
     evaluateCallInstr(command, context, agenda, stash)
+  },
+
+  [InstrType.LIST]: function (command: ListInstr, context: Context, agenda: Agenda, stash: Stash) {
+    const elements: es.Literal[] = []
+    for (let i = 0; i < command.arity; i++) {
+      elements.push(stash.pop().value)
```

Comment:
Can we keep the literals "boxed" up here still? We still want to keep their types in case we manipulate them in any way.

Unboxing to view in the final output shud be done at the end of `runECEMachine` when we are certain we are no longer manipulating the literals.

Reply:
👍 👍  added unwrapping of literals in list expression at the end of `runECEMachine`.. feels like its a bit hard coded..

Reply:
oh yes it is hahaha... it's temporary, but i've changed this in the next PR!! (i will update it to include list once ur PR gets merged)